helmfile/charts/networkpolicy-common/templates/kube-system/csi-cinder.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if and .Values.kubeSystem.enabled .Values.kubeSystem.openstack.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    {{- include "np.labels" . | nindent 4 }}
+  name: allow-csi-cinder-controller-plugin
+  namespace: kube-system
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app: csi-cinder-controllerplugin
+  ingress: []
```

Comment:
I see that you block traffic per np and not a default deny that we do for the other namespaces. Is there a reason for that?

Reply:
Yeah, I can add that if we want it but I'm a bit hesitant since it's the kube-system namespace. Then we would also need to wait for the DNS netpols to merge first.

Reply:
I thought as much. I would say go with your per np deny.